### DISCUSSION TITLE
Animated.View dimensions as shared variables

### DISCUSSION THREAD

**Discussion by ex3ndr** [software-mansion/react-native-reanimated#2172] [Ideas]
Being able to use `width` and `height` of a view in style could be insanely useful to adjust transformation based on current dimensions, for example - align view by it's center instead of top left corner. This will remove need for onLayout dancing.

**Comment by jgonet** [comment]
Doesn't [`measure()`](https://docs.swmansion.com/react-native-reanimated/docs/api/nativeMethods/measure) do that?

**Reply by ex3ndr** [reply]
Thanks to pointing to this! Will check. Docs are misleading showing usage only on react side.
